services/headless-lms/server/src/controllers/course_material/exercises.rs

```diff
@@ -362,6 +366,132 @@ async fn submit_peer_or_self_review(
     token.authorized_ok(web::Json(true))
 }
 
+/**
+ * POST `/api/v0/course-material/exercises/:exercise_id/flag-peer-review-answer - Post a report of an answer in peer review made by a student
+ */
+#[instrument(skip(pool))]
+async fn post_flag_answer_in_peer_review(
+    pool: web::Data<PgPool>,
+    payload: web::Json<NewFlaggedAnswer>,
+    user: AuthUser,
+) -> ControllerResult<web::Json<bool>> {
+    let mut conn = pool.acquire().await?;
+
```

Comment:
use givepeerreview claim here to verify the student actually got this answer to be peer reviewed